script.js

```diff
@@ -1,54 +1,54 @@
 // Questions will be asked
-const Questions = [
-  {
-    id: 0,
-    q: "1. Wydaje mi się, że docierają do mnie różne subtelne szczegóły otoczenia.",
-    a: [
-      { text: "tak", isCorrect: true },
-      { text: "nie", isCorrect: false },
-    ],
-  },
-  {
-    id: 1,
-    q: "2. Podlegam wpływowi nastrojów innych ludzi.",
-    a: [
-      { text: "tak", isCorrect: true },
-      { text: "nie", isCorrect: false },
-    ],
-  },
-  {
-    id: 2,
-    q: "3. Mam skłonność do dużej wrażliwości na ból.",
-    a: [
-      { text: "tak", isCorrect: true },
-      { text: "nie", isCorrect: false },
-    ],
-  },
-  {
-    id: 3,
-    q: "4. Jestem bardzo czuła na działanie kofeiny.",
-    a: [
-      { text: "tak", isCorrect: true },
-      { text: "nie", isCorrect: false },
-    ],
-  },
-  {
-    id: 4,
-    q: "5. W dni, kiedy dużo się dzieje, miewam potrzebę znalezienia chwili dla siebie, chowając się do łóżka, zaciemnionego pokoju czy gdziekolwiek indziej, gdzie mogę zaznać trochę spokoju i uciec od bodźców.",
-    a: [
-      { text: "tak", isCorrect: true },
-      { text: "nie", isCorrect: false },
-    ],
-  },
-  {
-    id: 5,
-    q: "6. Łatwo przytłaczają mnie takie bodźce jak ostre światło, silne zapachy, szorstkie tkaniny czy syreny przejeżdżających karetek.",
-    a: [
-      { text: "tak", isCorrect: true },
-      { text: "nie", isCorrect: false },
-    ],
-  },
-];
+const Questions = [{
+  id: 0,
+  q: "1. Wydaje mi się, że docierają do mnie różne subtelne szczegóły otoczenia.",
+  a: [{ text: "tak", isCorrect: true },
+  { text: "nie", isCorrect: false },
+  ]
+
+},
+{
+  id: 1,
+  q: "2. Podlegam wpływowi nastrojów innych ludzi.",
+  a: [{ text: "tak", isCorrect: true },
+  { text: "nie", isCorrect: false },
+  ]
+
+},
+{
+  id: 2,
+  q: "3. Mam skłonność do dużej wrażliwości na ból.",
+  a: [{ text: "tak", isCorrect: true },
+  { text: "nie", isCorrect: false },
+  ]
+
+},
+{
+  id: 3,
+  q: "4. Jestem bardzo czuła na działanie kofeiny.",
+  a: [{ text: "tak", isCorrect: true },
+  { text: "nie", isCorrect: false },
+  ]
+
+},
+{
+  id: 4,
+  q: "5. W dni, kiedy dużo się dzieje, miewam potrzebę znalezienia chwili dla siebie, chowając się do łóżka, zaciemnionego pokoju czy gdziekolwiek indziej, gdzie mogę zaznać trochę spokoju i uciec od bodźców.",
+  a: [{ text: "tak", isCorrect: true },
+  { text: "nie", isCorrect: false },
+  ]
+
+},
+{
+  id: 5,
+  q: "6. Łatwo przytłaczają mnie takie bodźce jak ostre światło, silne zapachy, szorstkie tkaniny czy syreny przejeżdżających karetek.",
+  a: [{ text: "tak", isCorrect: true },
+  { text: "nie", isCorrect: false },
+  ]
+
+}
+
+]
```

Comment:
This change is only whitespace. I used a code formatter for this, I didn't format this by hand, obviously :)